saleor/tests/fixtures.py

```diff
@@ -5664,6 +5664,26 @@ def page(db, page_type):
     return page
 
 
+@pytest.fixture
+def two_pages(page):
```

Comment:
Maybe `secound_page` instead of `two_pages` should be created here? 
Currently, this fixture is used by one test and is unwrapped at the start of the test.

Reply:
done